paddle/fluid/inference/tests/api/analyzer_dam_tester.cc

```diff
@@ -313,5 +322,44 @@ TEST(Analyzer_dam, compare_determine) {
                        input_slots_all);
 }
 
+// Save optim model
+TEST(Analyzer_resnet50, save_optim_model) {
```

Comment:
`Analyzer_resnet50` ? Do you want to name `Analyzer_dam`?

Reply:
Done, thanks~